python/libkvikio/libkvikio/_version.py

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import importlib.resources
+
+__version__ = (
+    importlib.resources.files("libkvikio").joinpath("VERSION").read_text().strip()
+)
+__git_commit__ = ""
```

Comment:
As part of the `rapids-build-backend` work, we've been updating all of these.  Can you please do this instead, to ensure that the `GIT_COMMIT` file that `rapids-build-backend` writes is read up into this constant?

```python
__version__ = (
    importlib.resources.files(__package__)
    .joinpath("VERSION")
    .read_text()
    .strip()
)
try:
    __git_commit__ = (
        importlib.resources.files(__package__)
        .joinpath("GIT_COMMIT")
        .read_text()
        .strip()
    )
except FileNotFoundError:
    __git_commit__ = ""

__all__ = ["__version__", "__git_commit__"]
```

example from `raft`: https://github.com/rapidsai/raft/blob/branch-24.08/python/pylibraft/pylibraft/_version.py